.editorconfig

```diff
@@ -1,161 +1,798 @@
-# editorconfig.org
-# Copied from the .NET Core repo https://github.com/dotnet/corefx/blob/master/.editorconfig
+# NOTE: Requires **VS2019 16.3** or later
 
-# top-most EditorConfig file
+# Microsoft SDL Roslyn Rules - Required and Recommended (Error) PreRelease version
+# Description: This rule set contains all Microsoft SDL required and recommended rules for Microsoft.CodeAnalysis.NetAnalyzers v5.0.3 (and Microsoft.CodeAnalysis.FxCopAnalyzers), configured to error rather than warn.  Generated 2021-02-08T19:40:15Z.
```

Comment:
This ruleset actually intended for Microsoft.CodeAnalysis.NetAnalyzers v5.0.3 as this one is designed for public consumption according to the Roslyn team. The "IAXXXX" rules are only used when the internal package is used but we do not need those at the moment. They plan on making that internal package public for opensource projects in the future. 

@bgavrilMS